jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  collectCoverageFrom: ['<rootDir>/src/**/*.{ts,js}'],
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/test/tsconfig.json',
```

Comment:
This was changed in newer version of ts-jest and will give you a warning

```suggestion
      tsconfig: '<rootDir>/test/tsconfig.json',
```